DigitalLearningSolutions.Data/Helpers/ExternalApis/LearningHubApiClient.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace DigitalLearningSolutions.Data.Helpers.ExternalApis

```

Comment:
I wouldn't put it in Helpers - I'd normally think of Helpers as static methods that help you out.
DigitalLearningSolutions.Data/ApiClients makes more sense to me.